nestkernel/simulation_manager.cpp

```diff
@@ -813,27 +842,6 @@ nest::SimulationManager::update_()
 }
```

Comment:
It's still in the update loop, it's inside of run/start_updating, in the same spot as simulate/resume